build.gradle

```diff
@@ -86,6 +86,10 @@ task integrationTest(type: Test) {
     classpath += sourceSets.integrationTest.runtimeClasspath
 }
 
+compileTestKotlin {
```

Comment:
I had to add this because I was getting:

> Task :compileTestKotlin FAILED
e: /Users/marcin.cebo/PubNub/code/java/src/test/java/com/pubnub/contract/objectV2/members/steps/WhenSteps.kt: (35, 40): Calls to static methods in Java interfaces are prohibited in JVM target 1.6. Recompile with '-jvm-target 1.8'

Reply:
ok